libraries/ghc-boot/changelog.md

```diff
@@ -0,0 +1,5 @@
+## 8.0.1  *May 2016*
+
+  * Bundled with GHC 8.0.1
+
+  * Initial version
```

Comment:
Probably better to omit this file altogether. It may create some confusion. Or replace the contents instead with the commit hash of the GHC repo as a version for `ghc-boot` so we can keep track as we integrate GHC 8 changes going forward.